id: PYSEC-0000-CVE-2016-1241
severity:
- type: CVSS_V3
  score: CVSS:3.0/AV:N/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N
details: Tryton 3.x before 3.2.17, 3.4.x before 3.4.14, 3.6.x before 3.6.12, 3.8.x
  before 3.8.8, and 4.x before 4.0.4 allow remote authenticated users to discover
  user password hashes via unspecified vectors.
affected:
- package:
    name: tryton
    ecosystem: PyPI
    purl: pkg:pypi/tryton
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: "0"
    - fixed: 3.2.17
references:
- type: REPORT
  url: https://bugs.tryton.org/issue5795
- type: ADVISORY
  url: http://www.debian.org/security/2016/dsa-3656
- type: ADVISORY
  url: http://www.tryton.org/posts/security-release-for-issue5795-and-issue5808.html
aliases:
- CVE-2016-1241
modified: "2016-09-08T19:06:00Z"
published: "2016-09-07T19:28:00Z"
